packages/express/src/providers/invoke-middleware.provider.ts

```diff
@@ -39,7 +40,7 @@ export class InvokeMiddlewareProvider implements Provider<InvokeMiddleware> {
   @config()
   protected defaultOptions: InvokeMiddlewareOptions = {
     chain: DEFAULT_MIDDLEWARE_CHAIN,
-    orderedGroups: ['cors', 'apiSpec', ''],
+    orderedGroups: ['cors', 'apiSpec', DEFAULT_MIDDLEWARE_GROUP],
```

Comment:
I don't see the key DEFAULT_MIDDLEWARE_GROUP (= 'middleware') being used anywhere else except in registerExpressMiddlewareInterceptor(). What does it indicate here, a separate chain ? or any/every middleware that was registered using registerExpressMiddlewareInterceptor() ?

Reply:
The group name is used to sort middleware within the same chain. This change is to make it explicit. So the picture can be:

1st chain (g1, g2, ...gm)
2nd chain (g1, g2, gn)